managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedLedgerImpl.java

```diff
@@ -3252,9 +3252,11 @@ public PositionImpl getPositionAfterN(final PositionImpl startPosition, long n,
                     totalEntriesInCurrentLedger = 0;
                 }
             } else {
-                totalEntriesInCurrentLedger = ledgers.get(currentLedgerId).getEntries();
+                totalEntriesInCurrentLedger = ledgers.get(currentLedgerId) != null
```

Comment:
We have to to the 'get' call only once, otherwise the second time you can get a null.

Reply:
+1

Reply:
Done